### PR TITLE
UI : Fix cypress for announcement

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- fix cypress for Announcement 

#
### Type of change :

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ describe("Entity Announcement", () => {
     cy.get('[data-testid="inActive-announcements"]').should('be.visible');
 
     // close announcement drawer
-    cy.get('.anticon > svg').should('be.visible').click();
+    cy.get('.anticon-close > svg > path').should('be.visible').click();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js:75]: Why we are dig in till `path` element?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js:75]: @chirag-madlani  As we have set onClick on SVG and it has a path as a child in it.  Either we can pass { multiple: true } or just go one step in and add an event listener there.

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js:75]:
```
cy.get('[data-testid="announcement-drawer"]').find('.anticon > svg').should('be.visible').click();
```

**Review by ShaileshParmar11 — COMMENTED** [review]
added comment

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js]: @ShaileshParmar11  Updated!

**Review by ShaileshParmar11 — APPROVED** [review]

**Comment by ShaileshParmar11** [comment]
not need now, fix is already merged in main

**Pull request closed without merging**